libs/execution/src/lib/types/valuetypes/visitors/is-valid-visitor.ts

```diff
@@ -39,7 +41,8 @@ export class IsValidVisitor extends ValuetypeVisitor<boolean> {
       );
     }
 
-    return !Number.isNaN(this.value);
+    const result = !Number.isNaN(this.value);
+    return result;
```

Comment:
Can be simplified again:
```suggestion
    return !Number.isNaN(this.value);
```